test/pages/LifeEvent.spec.js

```diff
@@ -271,16 +271,19 @@ describe("Life Event page", () => {
     })
     wrapper.vm.lifeEventBenefits = wrapper.vm.allLifeEventBenefits = [
       {
+        slug: "slugTwo",
```

Comment:
Is there any reason why `slugTwo` comes before `slugOne`?

Reply:
No